examples/rando/src/rando.cc

```diff
@@ -21,24 +19,55 @@ limitations under the License.
 
 #include "rando.h"
 
-using namespace Plugin;
+using Plugin::Config;
+using Plugin::ConfigPolicy;
+using Plugin::Metric;
+using Plugin::Meta;
+using Plugin::Type;
 
-Config::Policy Rando::getConfigPolicy() {
-  Config::Policy policy;
+ConfigPolicy Rando::get_config_policy() {
+  ConfigPolicy policy;
   return policy;
 }
 
-std::vector<Metric::Metric> Rando::getMetricTypes(Config::Config cfg) {
-  std::vector<Metric::Metric> metrics;
+std::vector<Metric> Rando::get_metric_types(Config cfg) {
+  std::vector<Metric> metrics;
```

Comment:
You could use http://en.cppreference.com/w/cpp/language/list_initialization instead of push_back below